stable/phabricator/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: phabricator
-version: 0.4.5
+version: 0.4.6
```

Comment:
@linki we avoid bumping the minor chart version whenever a previous chart deployment can be upgraded. We bump the minor version only if the chart contains updates that would break an existing deployment.